examples/nlp_finetuning/start_with_hpo.py

```diff
@@ -0,0 +1,53 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import boto3
+from syne_tune.backend.sagemaker_backend.sagemaker_utils import get_execution_role
+from syne_tune.config_space import choice, loguniform, uniform
+
+from renate.training import run_training_job
+
+config_space = {
+    "optimizer": "SGD",
+    "momentum": uniform(0.1, 0.9),
+    "weight_decay": 0.0,
+    "learning_rate": loguniform(1e-4, 1e-1),
+    "alpha": uniform(0.0, 1.0),
+    "batch_size": choice([16, 32]),
+    "memory_batch_size": 32,
+    "memory_size": 1000,
+    "loss_normalization": 0,
+    "loss_weight": uniform(0.0, 1.0),
+}
+
+if __name__ == "__main__":
+    AWS_ID = boto3.client("sts").get_caller_identity().get("Account")
+    AWS_REGION = "us-west-2"  # use your AWS preferred region here
+
+    run_training_job(
+        config_space=config_space,
+        mode="max",
+        metric="val_accuracy",
+        updater="ER",  # we train with Experience Replay
+        max_epochs=5,
+        # we select the first chunk of our dataset, you will probably not need this in practice
+        chunk_id=0,
+        config_file="renate_config.py",
+        requirements_file="../../requirements.txt",
```

Comment:
This works only as far as the example is in the right folder. 
You can create an ad-hoc requirements.txt in this folder such that it will be tied to the example.

Reply:
do we need the requirements file? besides the current problem with the avalanche dependency, there is no need for it, right?

Reply:
We need to make sure that all the dependencies are installed, right? Previously, we did this by adding a `requirements.txt` with `renate` listed. But that doesn't work if you're on `dev` and dependencies have changed compared to the PyPI version. I can copy the `requirements.txt` to the example folder, but then we need to make sure it stays in sync.